oak/proto/oak_api.proto

```diff
@@ -42,6 +42,8 @@ enum OakStatus {
   ERR_TERMINATED = 9;
   // Channel has no messages available to read.
   ERR_CHANNEL_EMPTY = 10;
+  // The node does not have sufficient permissions to perform the requested operation.
+  ERR_PERMISSION_DENIED = 11;
```

Comment:
I think it would be a good time to start adding some label/ERR_PERMISSION_DENIED info to ABI.md

Reply:
Good point, I added some notes there, but it does not quite make sense yet because the Wasm ABI does not even allow setting labels yet. Once we add those parameters in, then it should be expanded.